.lando.yml

```diff
@@ -0,0 +1,21 @@
+name: drupal.am
+recipe: drupal8
```

Comment:
We upgraded to drupal 9, please change the recipe, @iiinnnaaa

Reply:
Done @leo72